misk/src/main/kotlin/misk/tracing/TracingModule.kt

```diff
@@ -0,0 +1,16 @@
+package misk.tracing
+
+import misk.inject.KAbstractModule
+import misk.tracing.backends.jaeger.JaegerBackendModule
+
+class TracingModule(val config: TracingConfig) : KAbstractModule() {
+  override fun configure() {
+    if (config.tracer.equals("jaeger", true)) {
```

Comment:
Maybe just use the presence of a non-null `jaegar` property in `TracingConfig` to decide this vs having a separate `type` property?

Reply:
I like that. Should then verify only a single property is set